### ISSUE TITLE
Linux CUDA builds are failing to configure the system

### ISSUE THREAD

**Issue by malfet** [pytorch/pytorch#74968]
## Current Status
*ongoing*.

## Error looks like
All Xenial based builds are failing with:
```
Failed to fetch https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/Packages 404 Not Found
```

## Incident timeline (all times pacific)
*Include when the incident began, when it was detected, mitigated, root caused, and finally closed.*

<details>
<summary> Click for example </summary>

e.g.
- 10/30 7:27a incident began
- 10/30 8:30a detected by <method>
- 10/30 9:00 pm root caused as…
- 10/30 9:10 pm mitigated by…
- 10/31 10: am closed by…

</details>

## User impact
*How does this affect users of PyTorch CI?*

## Root cause
*What was the root cause of this issue?*

## Mitigation
*How did we mitigate the issue?*

## Prevention/followups
*How do we prevent issues like this in the future?*

**Comment by seemethere** [comment]
As a follow up we should probably just remove any instance of `apt-get update` or `apt-get install` inside of our CI scripts in general

**Comment by seemethere** [comment]
Issue appears to not be limited to 16.04 and is also applying to 18.04 as well

**Comment by malfet** [comment]
Yeah, looks like entire https://developer.download.nvidia.com/compute/machine-learning/repos/ is experiencing some some problems

**Comment by seemethere** [comment]
Also resolved with https://github.com/pytorch/builder/pull/1008

**Comment by seemethere** [comment]
Re-opening since other users may be experiencing this as well

**Comment by janeyx99** [comment]
Followup issue: https://github.com/pytorch/pytorch/issues/75703
